cpp/src/structs/utilities.hpp

```diff
@@ -66,7 +68,8 @@ std::tuple<table_view,
            std::vector<std::unique_ptr<column>>>
 flatten_nested_columns(table_view const& input,
                        std::vector<order> const& column_order,
-                       std::vector<null_order> const& null_precedence);
+                       std::vector<null_order> const& null_precedence,
+                       bool force_nullability_columns = false);
```

Comment:
Scoped enumerator for this? Maybe one already exists, i know there are a ton of `null` options

Reply:
I vote for adding an enum to this.